user/uploading-artifacts.md

```diff
@@ -80,6 +79,21 @@ addons:
 ```
 {: data-file=".travis.yml"}
 
+### Target Paths
+
+By default, artifacts will be uploaded to the path in the bucket
+defined by `/${TRAVIS_REPO_SLUG}/${TRAVIS_BUILD_NUMBER}/${TRAVIS_JOB_NUMBER}`.
+This value can be overridden by the `target_paths` option.
```

Comment:
I'd prefer something like.

You can change the upload path at build time using the `target_paths` key, for example:

Reply:
But regardless, :shipit: !

Ah, one question, `paths` plural?

Reply:
Yes. `target_paths` is plural.